### PR TITLE
statistics: add the updates of the new grammar

### PR DESCRIPTION
**Pull request by winoros**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

In 5.3, we introduced two new grammar and changed the default behavior of version stats. This pr is to add the docs for them.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: statistics.md [hunk 1]
```diff
@@ -73,21 +73,47 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
...
 ```
 
 - `WITH NUM BUCKETS` 用于指定生成直方图的桶数量上限。
 - `WITH NUM TOPN` 用于指定生成 TOPN 数目的上限。
 - `WITH NUM CMSKETCH DEPTH` 用于指定 CM Sketch 的长。
 - `WITH NUM CMSKETCH WIDTH` 用于指定 CM Sketch 的宽。
 - `WITH NUM SAMPLES` 用于指定采样的数目。
+- `WITH FLOAT_NUM SAMPLERATE` 用于指定采样率。
+
+##### `WITH NUM SAMPLES` 与 `WITH FLOAT_NUM SAMPLERATE`
```
Comment (TomShawn) [inline statistics.md:86]: 建议避免五级标题

File: statistics.md [hunk 2]
```diff
@@ -73,21 +73,47 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
...
+
+这两种设置对应了两种不同的收集采样的算法。`WITH NUM SAMPLES` 指定了采样集的大小，在 TiDB 中是以蓄水池采样的方式实现。`WITH FLOAT_NUM SAMPLERATE` 是在 v5.3.0 中新加入的采样方式，指定的是采样率的大小，是个取值范围 (0, 1] 的参数，在 TiDB 中是以伯努利采样的方式实现。伯努利采样的方式对比较大的表比较友好，在收集效率和资源使用上有很大提升。蓄水池采样由于中间结果集会产生一定的冗余结果因此会对内存等资源造成额外的压力，当表比较大的时候不推荐使用 `WITH NUM SAMPLES` 指定采样集的方式收集统计信息。
```
Comment (TomShawn) [inline statistics.md:88]:
```suggestion
`WITH NUM SAMPLES` 与 `WITH FLOAT_NUM SAMPLERATE` 这两种设置对应了两种不同的收集采样的算法。

- `WITH NUM SAMPLES` 指定了采样集的大小，在 TiDB 中是以蓄水池采样的方式实现。当表较大时，不推荐使用这种方式收集统计信息。因为蓄水池采样中间结果集会产生一定的冗余结果，会对内存等资源造成额外的压力。
- `WITH FLOAT_NUM SAMPLERATE` 是在 v5.3.0 中引入的采样方式，指定的采样率的大小，是取值范围 `(0, 1]` 的参数，在 TiDB 中是以伯努利采样的方式实现。这种方式更适合对较大的表进行采样，在收集效率和资源使用上更有优势。
```

File: statistics.md [hunk 3]
```diff
@@ -73,21 +73,47 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
...
+
+在 v5.3.0 之前 TiDB 总是采用蓄水池采样的方式收集统计信息。在 v5.3.0 开始的版本中，TiDB Version 2 的统计信息默认会选取伯努利采样的方式收集统计信息。可以通过 `WITH NUM SAMPLES` 语句来重新使用蓄水池采样的方式采样。
```
Comment (TomShawn) [inline statistics.md:90]:
```suggestion
在 v5.3.0 之前 TiDB 采用蓄水池采样的方式收集统计信息。自 v5.3.0 版本起，TiDB Version 2 的统计信息默认会选取伯努利采样的方式收集统计信息。若要重新使用蓄水池采样的方式采样，可以使用 `WITH NUM SAMPLES` 语句。
```

File: statistics.md (hunk 1 above)
Comment (TomShawn) [reply statistics.md:86]:
```suggestion
```

File: statistics.md [hunk 4]
```diff
@@ -73,21 +73,47 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
...
+
+> *注意：*
+>
+> 目前采样率是基于一个自适应的算法进行计算。当通过 [`SHOW STATS_META`](/sql-statements/sql-statement-show-stats-meta.md) 可以观察到一个表的行数时，我们会通过这个行数去计算采集 10 万行所对应的采样率。如果我们观察不到这个值，会通过表 [`TABLE_STORAGE_STATS`](/information-schema/information-schema-table-storage-stats.md) 的列 `TABLE_KEYS` 去作为另一个参考来计算采样率。
```
Comment (TomShawn) [inline statistics.md:94]:
```suggestion
> 目前采样率基于自适应算法进行计算。当你通过 [`SHOW STATS_META`](/sql-statements/sql-statement-show-stats-meta.md) 可以观察到一个表的行数时，可通过这个行数去计算采集 10 万行所对应的采样率。如果你观察不到这个值，可通过 [`TABLE_STORAGE_STATS`](/information-schema/information-schema-table-storage-stats.md) 表的 `TABLE_KEYS` 列作为另一个参考来计算采样率。
```

File: statistics.md [hunk 5]
```diff
@@ -73,21 +73,47 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
...
+> 通常情况下 `STATS_META` 相对 `TABLE_KEYS` 更可信，但是由于通过 [`TiDB Lightning`](/tidb-lightning/tidb-lightning-overview.md) 等方式导入时，`STATS_META` 在导入结束后的结果是 0。为了处理这个情况，我们在 `STATS_META` 的结果远小于 `TABLE_KEYS` 的结果时，也会去使用 `TABLE_KEYS` 去计算采样率。
```
Comment (TomShawn) [inline statistics.md:95]:
```suggestion
>
> 通常情况下，`STATS_META` 相对 `TABLE_KEYS` 更可信，但是通过 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 等方式导入数据结束后，`STATS_META`  结果是 `0`。为了处理这个情况，你可以在 `STATS_META` 的结果远小于 `TABLE_KEYS` 的结果时，使用 `TABLE_KEYS` 计算采样率。
```

File: statistics.md [hunk 6]
```diff
@@ -73,21 +73,47 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
...
+
+收集 TableName 中部分列的统计信息：
```
Comment (TomShawn) [inline statistics.md:97]:
```suggestion
以下语句收集 `TableName` 表中部分列的统计信息：
```

File: statistics.md [hunk 7]
```diff
@@ -73,21 +73,47 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
...
+
+{{< copyable "sql" >}}
+
+```sql
+ANALYZE TABLE TableName Columns [ColumnNameList] [WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH|SAMPLES]|[WITH FLOATNUM SAMPLERATE];
+```
+
+这个语法会收集所指定的列以及索引和扩展统计信息所涉及的列的统计信息。对于列数很多的表来说，需要统计信息的列可能只是一个很小的子集，通过这个语法我们可以极大的减轻收集统计信息的负担。
```
Comment (TomShawn) [inline statistics.md:105]:
```suggestion
这个语法会收集指定列以及索引的统计信息，以及扩展统计信息所涉及列的统计信息。如果表的列数较多，需要统计信息的列可能只是表很小的一个子集，通过这个语法可以极大地减轻收集统计信息的负担。
```

File: statistics.md [hunk 8]
```diff
@@ -73,21 +73,49 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
...
+- `WITH FLOAT_NUM SAMPLERATE` 用于指定采样率。
+
+`WITH NUM SAMPLES` 与 `WITH FLOAT_NUM SAMPLERATE` 这两种设置对应了两种不同的收集采样的算法。
+
+- `WITH NUM SAMPLES` 指定了采样集的大小，在 TiDB 中是以蓄水池采样的方式实现。当表较大时，不推荐使用这种方式收集统计信息。因为蓄水池采样中间结果集会产生一定的冗余结果，会对内存等资源造成额外的压力。
+- `WITH FLOAT_NUM SAMPLERATE` 是在 v5.3.0 中引入的采样方式，指定的采样率的大小，是取值范围 `(0, 1]` 的参数。在 TiDB 中是以伯努利采样的方式实现，更适合对较大的表进行采样，在收集效率和资源使用上更有优势。
+
+在 v5.3.0 之前 TiDB 采用蓄水池采样的方式收集统计信息。自 v5.3.0 版本起，TiDB Version 2 的统计信息默认会选取伯努利采样的方式收集统计信息。若要重新使用蓄水池采样的方式采样，可以使用 `WITH NUM SAMPLES` 语句。
+
+> *注意：*
```
Comment (TomShawn) [inline statistics.md:93]:
```suggestion
> **注意：**
```

File: statistics.md [hunk 9]
```diff
@@ -73,21 +73,49 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
...
+>
+> 目前采样率基于自适应算法进行计算。当你通过 [`SHOW STATS_META`](/sql-statements/sql-statement-show-stats-meta.md) 可以观察到一个表的行数时，可通过这个行数去计算采集 10 万行所对应的采样率。如果你观察不到这个值，可通过 [`TABLE_STORAGE_STATS`](/information-schema/information-schema-table-storage-stats.md) 表的 `TABLE_KEYS` 列作为另一个参考来计算采样率。
+>
+> 通常情况下，`STATS_META` 相对 `TABLE_KEYS` 更可信，但是通过 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 等方式导入数据结束后，`STATS_META`  结果是 `0`。为了处理这个情况，你可以在 `STATS_META` 的结果远小于 `TABLE_KEYS` 的结果时，使用 `TABLE_KEYS` 计算采样率。
```
Comment (TomShawn) [inline statistics.md:97]:
```suggestion
> 通常情况下，`STATS_META` 相对 `TABLE_KEYS` 更可信，但是通过 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 等方式导入数据结束后，`STATS_META` 结果是 `0`。为了处理这个情况，你可以在 `STATS_META` 的结果远小于 `TABLE_KEYS` 的结果时，使用 `TABLE_KEYS` 计算采样率。
```

File: statistics.md [hunk 10]
```diff
@@ -153,6 +181,12 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
 
 当某个表 `tbl` 的修改行数与总行数的比值大于 `tidb_auto_analyze_ratio`，并且当前时间在 `tidb_auto_analyze_start_time` 和 `tidb_auto_analyze_end_time` 之间时，TiDB 会在后台执行 `ANALYZE TABLE tbl` 语句自动更新这个表的统计信息。
 
+> *注意：*
```
Comment (TomShawn) [inline statistics.md:184]:
```suggestion
> **注意：**
```

File: statistics.md [hunk 11]
```diff
@@ -153,6 +181,12 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
...
+>
+> 目前自动更新无法记录手动 ANALYZE 时输入的配置项。因此当通过 WITH 语句控制 ANALYZE 的收集行为时，目前需要手动设置定时任务收集统计信息。
+
+[WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH]|[WITH NUM SAMPLES|WITH FLOATNUM SAMPLERATE];
+
```
Comment (TomShawn) [inline statistics.md:189]:
```suggestion
> 目前自动更新无法记录手动 ANALYZE 时输入的配置项。因此当通过 `WITH` 语句控制 ANALYZE 的收集行为时，目前需要手动设置定时任务收集统计信息。
>
> {{< copyable "sql" >}}

> ```sql
> [WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH]|[WITH NUM SAMPLES|WITH FLOATNUM SAMPLERATE];
> ```

```

File: statistics.md [hunk 12]
```diff
@@ -153,6 +181,16 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
 
 当某个表 `tbl` 的修改行数与总行数的比值大于 `tidb_auto_analyze_ratio`，并且当前时间在 `tidb_auto_analyze_start_time` 和 `tidb_auto_analyze_end_time` 之间时，TiDB 会在后台执行 `ANALYZE TABLE tbl` 语句自动更新这个表的统计信息。
 
+> **注意：**
+>
+> 目前自动更新无法记录手动 ANALYZE 时输入的配置项。因此当通过 `WITH` 语句控制 ANALYZE 的收集行为时，目前需要手动设置定时任务收集统计信息。
+>
+> {{< copyable "sql" >}}
+
```
Comment (TomShawn) [inline statistics.md:189]:
```suggestion
> 目前自动更新无法记录手动 ANALYZE 时输入的配置项。因此当通过 `WITH` 语句控制 ANALYZE 的收集行为时，目前需要手动设置定时任务收集统计信息。
>
> {{< copyable "sql" >}}
>
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: statistics.md [hunk 13]
```diff
@@ -73,21 +73,49 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
...
+```sql
+ANALYZE TABLE TableName Columns [ColumnNameList] [WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH]|[WITH NUM SAMPLES|WITH FLOATNUM SAMPLERATE];
+```
+
+这个语法会收集指定列以及索引的统计信息，以及扩展统计信息所涉及列的统计信息。如果表的列数较多，需要统计信息的列可能只是表很小的一个子集，通过这个语法可以极大地减轻收集统计信息的负担。
+
+> **注意：**
+>
+> ColumnNameList 不可为空。
```
Comment (xuyifangreeneyes) [inline statistics.md:111]: 这里需不需要强调一下是收集所有需要统计信息的列，而不是单独采集某一列统计信息的用法（跟 modify count 重置有关）

File: statistics.md [hunk 14]
```diff
@@ -97,15 +126,15 @@ IndexNameList 为空时会收集所有索引列的统计信息。
...
 ```
 
 收集 TableName 中所有的 PartitionNameList 中分区的索引列统计信息：
 
 {{< copyable "sql" >}}
 
 ```sql
-ANALYZE TABLE TableName PARTITION PartitionNameList INDEX [IndexNameList] [WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH|SAMPLES];
+ANALYZE TABLE TableName PARTITION PartitionNameList INDEX [IndexNameList] [WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH]|[WITH NUM SAMPLES|WITH FLOATNUM SAMPLERATE];
 ```
```
Comment (xuyifangreeneyes) [inline statistics.md:139]: 收集 TableName 中所有的 PartitionNameList 中分区的部分列统计信息：
```
ANALYZE TABLE TableName PARTITION PartitionNameList COLUMNS ColumnNameList [WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH]|[WITH NUM SAMPLES|WITH FLOATNUM SAMPLERATE];
```

File: statistics.md [hunk 15]
```diff
@@ -73,21 +73,50 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
...
+```sql
+ANALYZE TABLE TableName Columns [ColumnNameList] [WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH]|[WITH NUM SAMPLES|WITH FLOATNUM SAMPLERATE];
+```
+
+这个语法会收集指定列以及索引的统计信息，以及扩展统计信息所涉及列的统计信息。如果表的列数较多，需要统计信息的列可能只是表很小的一个子集，通过这个语法可以极大地减轻收集统计信息的负担。
+
+> **注意：**
+>
+> ColumnNameList 不可为空。
+> 该语法是全量收集的语法，如果遇到第一次收集了列 a 和 列 b 的统计信息之后想要增加列 c 的统计信息，需要同时指定三列 `ANALYZE table t columns a, b, c`，而不是 `ANALYZE TABLE t COLUMNS c`。
```
Comment (xuyifangreeneyes) [inline statistics.md:112]: 目前收集部分列统计信息只支持 tidb_analyze_version=2

File: statistics.md [hunk 16]
```diff
@@ -73,21 +73,50 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
...
+> 目前采样率基于自适应算法进行计算。当你通过 [`SHOW STATS_META`](/sql-statements/sql-statement-show-stats-meta.md) 可以观察到一个表的行数时，可通过这个行数去计算采集 10 万行所对应的采样率。如果你观察不到这个值，可通过 [`TABLE_STORAGE_STATS`](/information-schema/information-schema-table-storage-stats.md) 表的 `TABLE_KEYS` 列作为另一个参考来计算采样率。
+>
+> 通常情况下，`STATS_META` 相对 `TABLE_KEYS` 更可信，但是通过 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 等方式导入数据结束后，`STATS_META` 结果是 `0`。为了处理这个情况，你可以在 `STATS_META` 的结果远小于 `TABLE_KEYS` 的结果时，使用 `TABLE_KEYS` 计算采样率。
+
+以下语法收集 TableName 表中部分列的统计信息：
+
+{{< copyable "sql" >}}
+
+```sql
+ANALYZE TABLE TableName Columns [ColumnNameList] [WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH]|[WITH NUM SAMPLES|WITH FLOATNUM SAMPLERATE];
```
Comment (xuyifangreeneyes) [inline statistics.md:104]: ANALYZE TABLE TableName COLUMNS ColumnNameList
ColumnNameList 目前实现里是要求非空的，因此去掉 []

**Review by xuyifangreeneyes — COMMENTED** [review]
AnalyzeTableStmt ::=
    'ANALYZE' ( 'TABLE' ( TableNameList | TableName ( ( 'INDEX' IndexNameList | 'COLUMNS' ColumnNameList )? | 'PARTITION' PartitionNameList ( ( 'INDEX' IndexNameList | 'COLUMNS' ColumnNameList ) )? ) ) | 'INCREMENTAL' 'TABLE' TableName ( 'PARTITION' PartitionNameList )? 'INDEX' IndexNameList ) AnalyzeOptionListOpt

sql-statement-analyze-table.md 也需要更新一下 analyze 语法

**Comment by xuyifangreeneyes** [comment]
AnalyzeOptionListOpt ::=
    ( WITH AnalyzeOptionList )?

AnalyzeOptionList ::=
    AnalyzeOption ( ',' AnlyzeOption )*

AnalyzeOption ::=
    ( NUM ( 'BUCKETS' | 'TOPN' | ( 'CMSKETCH' ( 'DEPTH' | 'WIDTH' ) ) | 'SAMPLES' )  ) | ( FLOATNUM 'SAMPLERATE' )

ANALYZE 的语法图里关于 AnalyzeOptionListOpt 也可以具体说明一下

File: sql-statements/sql-statement-analyze-table.md [hunk 17]
```diff
@@ -16,13 +16,31 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-analyze-table/','/docs-cn/d
 
 ```ebnf+diagram
 AnalyzeTableStmt ::=
-    'ANALYZE' ( 'TABLE' ( TableNameList | TableName ( 'INDEX' IndexNameList | 'PARTITION' PartitionNameList ( 'INDEX' IndexNameList )? ) ) | 'INCREMENTAL' 'TABLE' TableName ( 'PARTITION' PartitionNameList )? 'INDEX' IndexNameList ) AnalyzeOptionListOpt
+    'ANALYZE' ( 'TABLE' ( TableNameList | TableName ( 'INDEX' IndexNameList? | 'COLUMNS' ColumnNameList | 'PARTITION' PartitionNameList ( 'INDEX' IndexNameList? | 'COLUMNS' ColumnNameList )? )? ) | 'INCREMENTAL' 'TABLE' TableName ( 'PARTITION' PartitionNameList )? 'INDEX' IndexNameList ) AnalyzeOptionListOpt
```
Comment (xuyifangreeneyes) [inline sql-statements/sql-statement-analyze-table.md:19]:
```suggestion
    'ANALYZE' ( 'TABLE' ( TableNameList | TableName ( 'INDEX' IndexNameList? | 'COLUMNS' ColumnNameList | 'PARTITION' PartitionNameList ( 'INDEX' IndexNameList? | 'COLUMNS' ColumnNameList )? )? ) | 'INCREMENTAL' 'TABLE' TableName ( 'PARTITION' PartitionNameList )? 'INDEX' IndexNameList? ) AnalyzeOptionListOpt
```

File: statistics.md [hunk 18]
```diff
@@ -97,15 +127,24 @@ IndexNameList 为空时会收集所有索引列的统计信息。
...
+ANALYZE TABLE TableName PARTITION PartitionNameList [WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH]|[WITH NUM SAMPLES|WITH FLOATNUM SAMPLERATE];
+```
+
+收集 TableName 中所有的 PartitionNameList 中分区的部分列统计信息：
+
+{{< copyable "sql" >}}
+
+```sql
+ANALYZE TABLE TableName PARTITION PartitionNameList COLUMNS ColumnNameList [WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH]|[WITH NUM SAMPLES|WITH FLOATNUM SAMPLERATE];
+
```
Comment (xuyifangreeneyes) [inline statistics.md:139]: remove the empty line

**Review by xuyifangreeneyes — APPROVED** [review]
LGTM

File: statistics.md [hunk 19]
```diff
@@ -73,21 +73,51 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
...
+```sql
+ANALYZE TABLE TableName COLUMNS ColumnNameList [WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH]|[WITH NUM SAMPLES|WITH FLOATNUM SAMPLERATE];
+```
+
+这个语法会收集指定列以及索引的统计信息，以及扩展统计信息所涉及列的统计信息。如果表的列数较多，需要统计信息的列可能只是表很小的一个子集，通过这个语法可以极大地减轻收集统计信息的负担。
+
+> **注意：**
+>
+> 该语法只支持 `tidb_analyze_version = 2` 的情况。
```
Comment (TomShawn) [inline statistics.md:111]:
```suggestion
> + 以上语法只支持 `tidb_analyze_version = 2` 的情况。
```

File: statistics.md (hunk 13 above)
Comment (TomShawn) [reply statistics.md:112]:
```suggestion
> + 在以上语法中，`ColumnNameList` 不可为空。
```

File: statistics.md (hunk 15 above)
Comment (TomShawn) [reply statistics.md:113]:
```suggestion
> + 以上语法是全量收集的语法。第一次收集了列 a 和 列 b 的统计信息之后，如果还想要增加列 c 的统计信息，需要在语法中同时指定三列 `ANALYZE table t columns a, b, c`，而不是只指定新增的那一列 `ANALYZE TABLE t COLUMNS c`。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**